File: Platform-Transport_Backend/PlatformTransport/src/main/java/com/platformtrasnport/platformtransport/dto/ReservationDto.java
package com.platformtrasnport.platformtransport.dto;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;

@Data
@Getter
@Setter
public class ReservationDto {
    private Long id;
    private LocalDate dateReservation;
    private Long employeId;
    private Long offreId;
    private float montant;
    private Long transactionId;
    private int nombrePlaces;
    private String pointDepart;
    private String destination;

}

File: Platform-Transport_FrontEnd/PlatformTransport/src/app/core/models/ReservationDto.ts
import { Employe } from "./Employe";

export class ReservationDto {
        id!: number;
        dateReservation!: Date;
        employeId!: number;
        offreId!: number;
        montant!: number;
        transactionId!: number | null;
        nombrePlaces!: number;
        pointDepart!: string;
        destination!: string;
}
